app/javascript/sdk/sdk.js

```diff
@@ -107,7 +107,11 @@ export const SDK_CSS = `
   width: auto !important;
 }
 
-.woot-widget-bubble.woot-widget--expanded img {
+.woot-widget-bubble.woot-widget--expanded.woot-widget-bubble-color--lighter div{
+  color: 	#37546D;
```

Comment:
Please move this as a variable since we are using this in 3 places

Reply:
Done